lib/resource/resource.ex

```diff
@@ -417,6 +425,27 @@ defmodule AshGraphql.Resource do
     end
   end
 
+  def encode_relay_id(%resource{} = record) do
+    primary_key = encode_primary_key(record)
+
+    "#{resource}:#{primary_key}"
```

Comment:
We should use the graphql type name as the prefix here, as opposed to the resource.